pkg/platform/model/vcs.go

```diff
@@ -675,89 +675,135 @@ func (cs indexedCommits) countBetween(first, last string) (int, error) {
 }
 
 // CommitPlatform commits a single platform commit
-func CommitPlatform(pj ProjectInfo, op Operation, name, version string, word int) error {
+func CommitPlatform(parentCommitID strfmt.UUID, op Operation, name, version string, word int) (*mono_models.Commit, error) {
```

Comment:
This can be dropped if we move the platform logic to also use ExecuteRequirementOperation.